src/Components/Web.Extensions/src/InputFile/InputFile.cs

```diff
@@ -73,6 +73,11 @@ internal async Task<IBrowserFile> ConvertToImageFileAsync(BrowserFile file, stri
         {
             var imageFile = await JSRuntime.InvokeAsync<BrowserFile>(InputFileInterop.ToImageFile, _inputFileElement, file.Id, format, maxWidth, maxHeight);
 
+            if (imageFile is null)
+            {
+                throw new InvalidOperationException("ToImageFile returned an unexpected null result.");
+            }
```

Comment:
I'm fine with this, but would also be fine with just coercing to a nonnull type since `InputFile.ts` will not return null, at least not with the current implementation.